workflows/utilities/wf_morgana_magic.wdl

```diff
@@ -0,0 +1,217 @@
+import "../../tasks/species_typing/betacoronavirus/task_pangolin.wdl" as pangolin
+import "../../tasks/species_typing/lentivirus/task_quasitools.wdl" as quasitools
+import "../../tasks/taxon_id/task_nextclade.wdl" as nextclade_task
+import "../utilities/wf_organism_parameters.wdl" as set_organism_defaults
+import "../utilities/wf_flu_track.wdl" as flu_track_wf
+
+workflow morgana_magic {
+  input {
+    String samplename
+    File assembly_fasta
+    File read1
+    File read2
+    String taxon_id
+    String seq_method
+    # consensus qc
+    Int? consensus_qc_cpu
+    Int? consensus_qc_disk_size
+    String? consensus_qc_docker
+    Int? consensus_qc_memory
+    # assembly metrics 
+    Int? assembly_metrics_cpu
+    Int? assembly_metrics_disk_size
+    String? assembly_metrics_docker
+    Int? assembly_metrics_memory
+    # flu track - irma
+    Int? irma_cpu
+    Int? irma_disk_size
+    String? irma_docker_image
+    Boolean? irma_keep_ref_deletions
+    Int? irma_memory
+    # flu track - genoflu
+    Int? genoflu_cpu
+    File? genoflu_cross_reference
+    Int? genoflu_disk_size
+    String? genoflu_docker
+    Int? genoflu_memory
+    # flu track - abricate
+    Int? abricate_flu_cpu
+    Int? abricate_flu_disk_size
+    String? abricate_flu_docker
+    Int? abricate_flu_memory
+    Int? abricate_flu_mincov
+    Int? abricate_flu_minid
+    # nextclade inputs
+    Int? nextclade_cpu
+    Int? nextclade_disk_size
+    String? nextclade_docker_image
+    Int? nextclade_memory
+    Int? nextclade_output_parser_cpu
+    Int? nextclade_output_parser_disk_size
+    String? nextclade_output_parser_docker
+    Int? nextclade_output_parser_memory
+    # pangolin inputs
+    Int? pangolin_cpu
+    Int? pangolin_disk_size
+    String? pangolin_docker_image
+    Int? pangolin_memory
+  }
+  call set_organism_defaults.organism_parameters {
+    input:
+      taxon_id = taxon_id,
+      organism = "unsupported",
+      pangolin_docker_image = pangolin_docker_image
+  }
+  if (organism_parameters.standardized_organism != "unsupported") { # occurs in theiameta_panel
+    call consensus_qc_task.consensus_qc {
+      input:
+        assembly_fasta = assembly_fasta,
+        reference_genome = organism_parameters.reference,
+        genome_length = organism_parameters.genome_length,
+        cpu = consensus_qc_cpu,
+        disk_size = consensus_qc_disk_size,
+        docker = consensus_qc_docker,
+        memory = consensus_qc_memory
+    }
+  }
+  if (organism_parameters.standardized_organism == "flu") {
+    call flu_track_wf.flu_track {
+      input:
+        samplename = samplename,
+        read1 = read1,
+        read2 = read2,
+        seq_method = seq_method,
+        standardized_organism = organism_parameters.standardized_organism,
+        analyze_flu_antiviral_substitutions = false, # don't try to look for antiviral substitutions?? or maybe? not sure
```

Comment:
I think leaving this as false, for now, makes sense. We can expand it further down the road, if so desired.